### ISSUE TITLE
CurImagePlugin and Transparency

### ISSUE THREAD

**Issue by wiredfool** [python-pillow/Pillow#807]
```
from PIL Import Image
im = Image.open('Tests/images/deerstalker.cur')
im.save('cur.png')
```
```
# shows transparency
% display Tests/images/deerstalker.cur 
#no transparency
% display cur.png 
```

**Comment by radarhere** [comment]
Looks like this was fixed by #1125

**Comment by hugovk** [comment]
Tested per https://github.com/python-pillow/Pillow/issues/807#issue-38099388 and cur.png now has transparency.

Closing, thanks for following up!
